quasar-utils/src/main/groovy/net/corda/plugins/quasar/QuasarPlugin.groovy

```diff
@@ -120,12 +120,16 @@ class QuasarPlugin implements Plugin<Project> {
         return configuration
     }
 
-    private static Configuration createCompileOnlyConfiguration(String name, ConfigurationContainer configurations) {
+    private static Configuration createCompileConfiguration(String name, ConfigurationContainer configurations) {
+        return createCompileConfiguration(name, "Implementation", configurations)
```

Comment:
you might want to use `org.gradle.api.plugins.JavaPlugin#IMPLEMENTATION_CONFIGURATION_NAME`

Reply:
I suspect that value is "implementation" rather than "Implementation". I am leveraging Gradle's naming convention to modify all `xxxxImplementation` configurations here, where `xxxx` could be `test`, `integrationTest`, `smokeTest`, `bongoTest` etc.